docs/Tutorial.md

```diff
@@ -478,7 +478,7 @@ import { List, Datagrid, TextField, ReferenceField } from "react-admin";
 export const PostList = () => (
   <List>
     <Datagrid rowClick="edit">
-      <ReferenceField source="userId" reference="users" />
+      <ReferenceField source="userId" reference="users" link="show" />
```

Comment:
No, this is too early. Readers will wonder why you add that.

Reply:
Removed and added below